.chloggen/jm-prom-rw-exporter-add-support-for-rw2.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: prometheusremotewriteexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add `exporter.prometheusremotewritexporter.enableSendingRW2` feature gate to and configure the exporter to send Prometheus remote write 2.0 version.
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [33661]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: As the the support for rw2 is not fully implemented (this is only the initial implementation) we only recommend testing this.
```

Comment:
Would welcome feedback on the subtext. Basically I feel like  we should call out the partial implementation and that people should only test it, not use it in production since it's not ready but also lot of things are not implemented.

Reply:
You can keep it simpler: PRW 2.0 support for the exporter is still under development.